project/plugins.sbt

```diff
@@ -1,9 +1,11 @@
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "4.1.0")
 addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "1.4.0")
-addSbtPlugin("com.dwijnand" % "sbt-dynver" % "2.0.0")
+addSbtPlugin("com.dwijnand" % "sbt-dynver" % "3.0.0+4-1be64920")
 addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.3.2")
 addSbtPlugin("com.eed3si9n" % "sbt-unidoc" % "0.4.1")
 addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "2.0.2")
 addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.2")
+addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.3.0")
 addSbtPlugin("com.lightbend" % "sbt-whitesource" % "0.1.10")
 addSbtPlugin("lt.dvim.paradox" % "sbt-paradox-local" % "0.1")
+resolvers += Resolver.bintrayIvyRepo("2m", "sbt-plugins")
```

Comment:
This should not be needed. alpakka-kafka project has it because of the custom version of docker-compose plugin.